pkg/cluster/mpi/mpi.go

```diff
@@ -11,9 +11,9 @@ import (
 
 const (
 
-	// SSH port used by MPI and a name of this port within the service
+	// SSH port used by MPI worker
 	sshdPort     = 2222
-	sshdPortName = "sshd"
+	sshdPortName = "sshd" // service port name
```

Comment:
Can you remove this comment? It will cause a merge conflict with my PR (the service name is not Istio compatible).